### PR TITLE
Don't change hybrid file system type

### PR DESCRIPTION
**Pull request by e9925248**

### CODE REVIEW

File: modules/KIWILinuxRC.sh [hunk 1]
```diff
@@ -9712,6 +9700,35 @@ function reloadKernel {
...
+        e2fsck -p $device
+    elif [ "$FSTYPE" = "btrfs" ];then
+        btrfsck $device
+    elif [ "$FSTYPE" = "xfs" ];then
+        xfs_repair -n $device
+    else
+        FSTYPE=$FSTYPE_SAVE
+        # don't know how to resize this filesystem
+        Echo "Don't know how to resize ${FSTYPE}... skip it"
+        return
```
Comment (schaefi) [inline modules/KIWILinuxRC.sh]: the error message is misleading, guess this was just a copy & paste typo :)

**Comment by schaefi** [comment]
The overall change looks good to me, thanks much

**Comment by e9925248** [comment]
Checked the two texts. fat [= container with btrfs inside] is probably not handled correctly.

**Comment by schaefi** [comment]
Thanks, merging

**Pull request merged**